### DISCUSSION TITLE
How to validate multi dimensional array with DTO

### DISCUSSION THREAD

**Discussion by pprev94** [symfony/symfony#51810] [Q&A]
Hi,

In my request, I've got variable $table which is a multi dimensional array :
["table1" => ["A","B","C"], "table2" => ["D","E","F"]]

How to validate this variable by a DTO ?

Thanks

**Comment by MatTheCat** [comment]
Hi,

I don’t understand what you’re asking.

Anyways you would use a [validator](https://symfony.com/doc/current/validation.html) to validate an array or/and a DTO.
